src/gluonts/model/n_beats/_ensemble.py

```diff
@@ -296,8 +296,8 @@ class NBEATSEnsembleEstimator(Estimator):
         If type is "T" (trend), then it corresponds to the degree of the polynomial.
         If the type is "S" (seasonal) then its not used.
         A list of ints of length 1 or 'num_stacks'.
-        Default and recommended value for generic mode: [2]
-        Recommended value for interpretable mode: [2]
+        Default and recommended value for generic mode: [32]
+        Recommended value for interpretable mode: [3]
```

Comment:
A minor comment here: 32 should mentioned just as “default” (not “recommended”), since it’s not mentioned in the paper, and 3 if I understand correctly is also “default” (other than “recommended”). Similarly in the other affected modules.

Reply:
Actually, the polynomial degree of 2 is mentioned in the paper (to get that the parameter needs to be 3 though, not 2). The rest is True.